2-projects/1-todo-list/script.js

```diff
@@ -1,6 +1,30 @@
+let list = document.getElementById("todo-list");
+
+while (list.hasChildNodes()) {
+  list.removeChild(list.firstChild);
+}
+
 function populateTodoList(todos) {
-  let list = document.getElementById("todo-list");
+  // let list = document.getElementById("todo-list");
   // Write your code to create todo list elements with completed and delete buttons here, all todos should display inside the "todo-list" element.
+  todos.forEach((todo) => {
+    let li = document.createElement("li");
+    li.className =
+      "list-group-item d-flex justify-content-between align-items-center";
+
+    let text = document.createTextNode(todo.task);
+
+    //line-through and delete buttons
+    let span = document.createElement("span");
+    span.className = "badge bg-primary rounded-pill";
+    span.innerHTML = `
```

Comment:
rather than using innerHTML try and create elements like you have elsewhere, you can do this for i tags too by doing:
` let trashIcon = document.createElement("i");`
` let checkIcon = document.createElement("i");`